src/sections/dataset/dataset-files/dataset-upload-files-button/DatasetUploadFilesButton.tsx

```diff
@@ -4,15 +4,17 @@ import { useSession } from '../../../session/SessionContext'
 import styles from './DatasetUploadFilesButton.module.scss'
 import { useTranslation } from 'react-i18next'
 import { useDataset } from '../../DatasetContext'
+import { useNotImplementedModal } from '../../../not-implemented/NotImplementedModalContext'
 
 export function DatasetUploadFilesButton() {
```

Comment:
When I click the Upload Files button in the storybook I get this message. This is not happening with the rest of the buttons implementing the new modal, can you try it?

<img width="1014" alt="image" src="https://github.com/IQSS/dataverse-frontend/assets/23359572/283eb76a-20c7-416f-8fb8-76bb81ae9c11">

Reply:
I'm not sure what happened, it worked for me - this is what I see when I click Upload Files. Can you try it again?  Maybe you are trying another Story?
<img width="1348" alt="Screenshot 2023-11-27 at 7 42 15 PM" src="https://github.com/IQSS/dataverse-frontend/assets/675224/873a48c8-9181-4c17-ba93-4676f82c2ec3">